src/status_im/ui/screens/browser/events.cljs

```diff
@@ -71,3 +71,11 @@
  (fn [{:keys [db]} [browser-id]]
    {:db (update-in db [:browser/browsers] dissoc browser-id)
     :data-store/remove-browser browser-id}))
+
+(handlers/register-handler-fx
+ :handle-browser-error
+ [re-frame/trim-v]
+ (fn [{:keys [db]} [browser-id]]
+   {:dispatch-n [[:navigate-back]
```

Comment:
in this case user doesn't see error, let's leave him on the error page

Reply:
@flexsurfer can we just show the error in an alert instead of rendering the error in the browser? this is how it behaves now.

Reply:
@psdp yes it's fine, but user should stay on browser screen